system/modules/core/modules/ModuleLogin.php

```diff
@@ -76,7 +76,8 @@ public function generate()
 				// Redirect to the jumpTo page
 				if ($this->jumpTo && ($objTarget = $this->objModel->getRelated('jumpTo')) !== null)
```

Comment:
A general idea. If we would write `($objTarget = $this->objModel->getRelated('jumpTo')) instanceof \PageModel`, it would be more safe, we would not need the `@var` hint and work exactly the same.